packages/table/src/Table.js

```diff
@@ -59,30 +87,94 @@ class Table extends React.Component {
                 column.props &&
                 this._renderColumnTitle(index, column.props)
             )}
+            {collapsed && this._renderColumnTitle()}
           </tr>
         </thead>
-        <tbody className="body">
-          {data.map((row, index) => {
-            return (
-              <tr key={index} className="row">
-                {selectableRows && this._renderSelectRowButton(index)}
-                {React.Children.map(
-                  children,
-                  column =>
-                    column &&
-                    column.props &&
-                    this._renderColumnCell(row, index, column.props)
-                )}
-              </tr>
-            );
-          })}
-        </tbody>
+        {data.map((row, index) => {
+          const isRowGroup =
+            row[rowGroupKey] && Array.isArray(row[rowGroupKey]);
+          if (isRowGroup) {
+            return this._renderRowGroup(row, index);
+          }
+          return (
+            <tbody key={index} className="body">
+              {this._renderRow(row, index)}
+            </tbody>
+          );
+        })}
       </StyledTable>
     );
   }
 
+  _renderRow = (row, index, subIndex = "", group = false) => {
+    const { selectableRows, collapsed, children } = this.props;
+    const rowKey = `${index},${subIndex}`;
+    const selected = this.state.selectedRows.indexOf(rowKey) !== -1;
```

Comment:
I prefer`includes` over `indexOf` because it's declarative.

A downside is that it's not supported by IE.
We could add an indication of this restriction in the readme, so the use of a polyfill would be decided at the application using this component.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Anyway, this is minor and I'm OK if we decide to keep `indexOf`.